[2023-03-24T21:13:01.315+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test-tutorial.print_date scheduled__2023-03-24T21:12:00+00:00 [queued]>
[2023-03-24T21:13:01.326+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test-tutorial.print_date scheduled__2023-03-24T21:12:00+00:00 [queued]>
[2023-03-24T21:13:01.326+0000] {taskinstance.py:1282} INFO - 
--------------------------------------------------------------------------------
[2023-03-24T21:13:01.326+0000] {taskinstance.py:1283} INFO - Starting attempt 1 of 2
[2023-03-24T21:13:01.326+0000] {taskinstance.py:1284} INFO - 
--------------------------------------------------------------------------------
[2023-03-24T21:13:01.343+0000] {taskinstance.py:1303} INFO - Executing <Task(BashOperator): print_date> on 2023-03-24 21:12:00+00:00
[2023-03-24T21:13:01.348+0000] {standard_task_runner.py:55} INFO - Started process 3193 to run task
[2023-03-24T21:13:01.351+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'test-tutorial', 'print_date', 'scheduled__2023-03-24T21:12:00+00:00', '--job-id', '50', '--raw', '--subdir', 'DAGS_FOLDER/test-tutorial.py', '--cfg-path', '/tmp/tmp3flyfkc7']
[2023-03-24T21:13:01.351+0000] {standard_task_runner.py:83} INFO - Job 50: Subtask print_date
[2023-03-24T21:13:01.425+0000] {task_command.py:388} INFO - Running <TaskInstance: test-tutorial.print_date scheduled__2023-03-24T21:12:00+00:00 [running]> on host e7c16d9fd17d
[2023-03-24T21:13:01.519+0000] {taskinstance.py:1511} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=daniel@example.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=test-tutorial
AIRFLOW_CTX_TASK_ID=print_date
AIRFLOW_CTX_EXECUTION_DATE=2023-03-24T21:12:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-03-24T21:12:00+00:00
[2023-03-24T21:13:01.520+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-03-24T21:13:01.521+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'date +%Y-%m-%d %H:%M:%S']
[2023-03-24T21:13:01.531+0000] {subprocess.py:86} INFO - Output:
[2023-03-24T21:13:01.534+0000] {subprocess.py:93} INFO - date: extra operand ‘%H:%M:%S’
[2023-03-24T21:13:01.534+0000] {subprocess.py:93} INFO - Try 'date --help' for more information.
[2023-03-24T21:13:01.534+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2023-03-24T21:13:01.547+0000] {taskinstance.py:1775} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 197, in execute
    f"Bash command failed. The command returned a non-zero exit code {result.exit_code}."
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2023-03-24T21:13:01.552+0000] {taskinstance.py:1326} INFO - Marking task as UP_FOR_RETRY. dag_id=test-tutorial, task_id=print_date, execution_date=20230324T211200, start_date=20230324T211301, end_date=20230324T211301
[2023-03-24T21:13:01.566+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 50 for task print_date (Bash command failed. The command returned a non-zero exit code 1.; 3193)
[2023-03-24T21:13:01.602+0000] {local_task_job.py:212} INFO - Task exited with return code 1
[2023-03-24T21:13:01.637+0000] {taskinstance.py:2585} INFO - 0 downstream tasks scheduled from follow-on schedule check
